test/parallel/test-buffer-read.js

```diff
@@ -52,21 +52,52 @@ read(buf, 'readUIntBE', [2, 2], 0x48ea);
 read(buf, 'readUIntLE', [2, 2], 0xea48);
 
 // Attempt to overflow buffers, similar to previous bug in array buffers
-assert.throws(() => Buffer.allocUnsafe(8).readFloatBE(0xffffffff),
-              RangeError);
-assert.throws(() => Buffer.allocUnsafe(8).readFloatLE(0xffffffff),
-              RangeError);
+assert.throws(
+  () => Buffer.allocUnsafe(8).readFloatBE(0xffffffff),
+  {
```

Comment:
The following object can be stored in a variable and can be passed as a second argument:
```
{
   name: 'RangeError [ERR_OUT_OF_RANGE]'
}
```

Same applies to this object on line 89 and 96:
```
{
      name: 'RangeError [ERR_BUFFER_OUT_OF_BOUNDS]',
      message: 'Attempt to write outside buffer bounds'
 }
```

Reply:
Thanks @trivikr I will do the changes and commit.

Reply:
I see I missed the point. I had to parametrize the object, not the value. Pushing the changes in next commit.